arbnode/seq_coordinator.go

```diff
@@ -513,6 +547,7 @@ func (c *SeqCoordinator) update(ctx context.Context) time.Duration {
 			return c.retryAfterRedisError()
 		}
 		log.Info("caught chosen-coordinator lock")
+		atomic.StoreInt32(&c.initialDelayedSequencing, 0)
 		c.sequencer.DontForward()
```

Comment:
This deserves some thought.. but the initialDelayedSequencing is probably not needed, since the sequencer won't try sequencing anything until DontForward is called.. maybe just need to take care of start-up (e.g. by default have the sequencer forward to /dev/null till coordinator says otherwise)

Reply:
Transient errors causing us to become not-chosen then chosen are probably o.k even if we don't force-sequence all delayed messages sent during the transient error. In any other case, sequencer should be forwarding until we get here.

Reply:
Re: startup, forwarding to /dev/null would mean dropping txs on startup which doesn't seem ideal. I think currently we'll hold txs in the queue until the sequencer figures out who's chosen.